static-assets/components/cstudio-common/common-api.js

```diff
@@ -160,17 +160,17 @@ var nodeOpen = false,
          */
```

Comment:
**Specifically** for common API: Revert the changes, add back what ever changes for this PR, disable the "reformat code" option prior to committing and then commit. Basically make sure only the changes of this PR come up here without all the code reformat (for this file, we need to do that on a stand-alone PR some other time).

Reply:
Still not done.

Reply:
done, please verify